mono/metadata/exception.c

```diff
@@ -403,6 +521,12 @@ mono_get_exception_invalid_operation (const char *msg)
 					"InvalidOperationException", msg);
 }
 
+MonoExceptionHandle
+mono_exception_new_invalid_operation (const char *msg, MonoError *error)
```

Comment:
I'm not sure this is needed:
Any remaining (non-API) callers of `mono_get_exception_invalid_operation` should be converted to use `mono_error_set_invalid_operation` instead.

Likewise for the other exceptions.

Reply:
Every function I actually implemented does have an analogous user that in my other clone I have converted and hope to commit soon. (vs. the declarations in the header, of which there are extra). Should I make the extra change now or hold off? I'm inclined to hold off.

Reply:
Actually it does not appear to easy to replace exception with error in the cases I was converting. There is stuff like setting of nested exceptions -- that that I'm not sure errors support. But anyway we shouldn't do that yet.

Reply:
Good point - re nested exceptions - that's not currently possible with MonoError.